src/libraries/System.Runtime.Caching/Directory.Build.props

```diff
@@ -6,6 +6,8 @@
          to a different assembly. -->
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
```

Comment:
you can remove the `IncludePlatformAttributes` from src/libraries/System.Runtime.Caching/src/System.Runtime.Caching.csproj#L5 since you're adding it in the Directory.Build.props.